src/Explore/Views/map.twig

```diff
@@ -15,7 +15,7 @@
         <img class='dataIcon' id='crimeIcon' src='images/fingerprint.png' title='Crime Data by Severity' data-toggle='tooltip' data-placement='right'></img>
         <img class='dataIcon' id='schoolIcon' src='images/apple.png' title='Schools by Rating' data-toggle='tooltip' data-placement='right'></img>
         <!--<img class='dataIcon' id='foodIcon' src='images/food.png' title='Health Inspection Data' data-toggle='tooltip' data-placement='right'></img>-->
-        <img class='dataIcon' id='housingIcon' src='images/housingCost.png' title='Housing Market Data' data-toggle='tooltip' data-placement='right'></img>
+        <!--<img class='dataIcon' id='housingIcon' src='images/housingCost.png' title='Housing Market Data' data-toggle='tooltip' data-placement='right'></img>-->
```

Comment:
I'll remove the housing one since we don't actually have an endpoint for that, but I'm going to leave the food one so it can easily be added back once OHI gets back up and running